README.md

```diff
@@ -1,5 +1,54 @@
-# 2dTo3dVector
-## Additions
-A CLI to convert 2D vector data to 3D vector data. 
-Testing a commit with VSCode
-Now, submitting a Pull Request instead of a Commit
\ No newline at end of file
+# Mountaineer
+A CLI to convert 2D vector data to 3D vector data.  
+
+## Usage
+CLI Usage: One liner
+
+mountaineer [input file] [lat] [lon] [output file]
+
+[input file]: Input file. File type can be .KML, .SHP, .GeoJSON (preferred). These are geospatial formats that contain Lat/long coordinates. Input file length should not exceed 128 characters. 
+
+[lat]: Latitude field. Required to create WKT
+[lon]: Longitude field. Required to create WKT
```

Comment:
Why are these required? I thought the coords would be pulled from the input file?

Reply:
Upon retrospection, these are no longer required. Geometry of the Geojson will be pulled and the fields will not need to be specified.